src/families/ethereum/synchronisation.ts

```diff
@@ -296,6 +323,136 @@ const txToOps =
 
           return all;
         });
+
+    // Creating NFTOps from transfer events related to ERC721 only
+    const erc721Operations = !erc721_transfer_events
+      ? []
+      : flatMap(erc721_transfer_events, (event) => {
+          const sender = safeEncodeEIP55(event.sender);
+          const receiver = safeEncodeEIP55(event.receiver);
+          const contract = safeEncodeEIP55(event.contract);
+          const tokenId = event.token_id;
+          const nftId = encodeNftId(id, event.contract, tokenId);
+          const sending = addr === sender;
+          const receiving = addr === receiver;
+
+          if (!sending && !receiving) {
+            return [];
+          }
+
+          const all: NFTOperation[] = [];
+
+          if (sending) {
+            const type = "OUT";
+            all.push({
+              id: `${nftId}-${hash}-${type}`,
```

Comment:
make sure the ending op Id includes the accountID as we do and keep the order maybe like 
accountID hash nftId type
So we can still parse it easily or maybe nftId at the end to debate :thinking:

Reply:
Will do 👌